function/random/random.js

```diff
@@ -4,49 +4,63 @@ module.exports = function(RED) {
     function RandomNode(n) {
         RED.nodes.createNode(this,n);
 
-        this.low  = Number(n.low) || ""; 
-        this.high = Number(n.high) || "";
+        this.status({}); // blank status on a deploy
```

Comment:
Not sure we need status for this sort of thing. Maybe use hoverover on output pin.